README.md

```diff
@@ -7,7 +7,7 @@
 [![Code Quality Checks](https://github.com/lmc-eu/spirit-design-system/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/lmc-eu/spirit-design-system/actions)
 [![Coverage Status](https://coveralls.io/repos/github/lmc-eu/spirit-design-system/badge.svg?branch=main)](https://coveralls.io/github/lmc-eu/spirit-design-system?branch=main)
 
-Spirit is an open-source design system developed by [LMC][lmc].
+Spirit is an open-source design system developed by [Alma Career][alma-career].
```

Comment:
this feels weird :)) Alma career link to lmc-eu organisation. 

how about something like this?

```suggestion
Spirit is an open-source design system developed by [Alma Career (previously called LMC)][alma-career].
```

Reply:
And then when (if) we move to the alma organisation we can remove the bracketed part.

Reply:
Ok, that sounds reasonable.